src/core_plugins/vega/public/vega_editor_template.html

```diff
@@ -0,0 +1,75 @@
+<div class="vegaEditor" ng-controller="VegaEditorController">
+  <div
+    ui-ace="{
+      onLoad: aceLoaded,
+      useWrapMode: true,
+      advanced: {
+        highlightActiveLine: false
+      },
+      rendererOptions: {
+        showPrintMargin: false
+      },
+      mode: 'hjson'
+    }"
+    id="vegaAceEditor"
+    ng-model="vis.params.spec"
+    data-test-subj="vega-editor"
+  ></div>
+
+  <div id="editor_actions">
+    <span dropdown>
+      <button
+        id="vegaHelp"
+        class="editor_action"
+        dropdown-toggle href="#"
+        aria-label="Vega help"
+      >
+        <span class="kuiIcon fa-question-circle"></span>
+      </button>
+      <ul
+        class="dropdown-menu"
+        role="menu"
+        aria-labelledby="vegaHelp"
+      >
+        <li role="menuitem">
+          <a target="_blank" href="https://www.elastic.co/guide/en/kibana/master/vega-graph.html">
```

Comment:
I think, we shouldn't point to docs at `master`, but rather to a specific version. We should either link to `current` or even better - if we have the possibilities for that - link to the actual Kibana version that's running. @epixa any possibilities to do that?

Reply:
@timroes You're right.  There exists a documentation link service, which is really rudimentary, but it handles the docs branch logic.  I think we're using it for the discover query bar docs link, if you need an example.

Reply:
@epixa are you referring to https://github.com/elastic/kibana/blob/d22f4eec0bc8616f0ed03659e8c246031fd152b1/src/core_plugins/kibana/public/context/app.html#L48 ?  If so, how does `contextApp.state.navigation.discover.url` get resolved - is there anything with "discover" keyword registration? Thx!